actionmailer/lib/action_mailer/test_helper.rb

```diff
@@ -89,8 +89,16 @@ def assert_no_emails(&block)
     #       ContactMailer.welcome.deliver_later
     #     end
     #   end
-    def assert_enqueued_emails(number, &block)
-      assert_enqueued_jobs number, only: [ ActionMailer::DeliveryJob, ActionMailer::Parameterized::DeliveryJob ], &block
+    #
+    # It can be asserted that jobs of a specific kind are enqueued.
+    #
+    #   def test_emails
+    #     assert_enqueued_emails only: MyCustomDeliveryJob do
```

Comment:
Can you add a `1` for the number of enqueued emails here, just so that way it matches the rest of the prior examples.